bundle/manifests/nic-configuration-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,373 @@
+          - create
+          - delete
+          - get
+          - update
+        - apiGroups:
+          - configuration.net.nvidia.com
+          resources:
+          - nicconfigurationtemplates
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - configuration.net.nvidia.com
+          resources:
+          - nicconfigurationtemplates/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - configuration.net.nvidia.com
+          resources:
+          - nicconfigurationtemplates/status
+          verbs:
+          - get
+          - patch
+          - update
+        - apiGroups:
+          - configuration.net.nvidia.com
+          resources:
+          - nicdevices
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - configuration.net.nvidia.com
+          resources:
+          - nicdevices/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - configuration.net.nvidia.com
+          resources:
+          - nicdevices/status
+          verbs:
+          - get
+          - patch
+          - update
+        - apiGroups:
+          - coordination.k8s.io
+          resources:
+          - leases
+          verbs:
+          - create
+          - get
+          - update
+        - apiGroups:
+          - maintenance.nvidia.com
+          resources:
+          - nodemaintenances
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - security.openshift.io
+          resourceNames:
+          - privileged
+          resources:
+          - securitycontextconstraints
+          verbs:
+          - use
+        serviceAccountName: nic-configuration-operator-controller-manager
+      deployments:
+      - label:
+          app.kubernetes.io/managed-by: kustomize
+          app.kubernetes.io/name: nic-configuration-operator
+          control-plane: controller-manager
+        name: nic-configuration-operator-controller-manager
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              control-plane: controller-manager
+          strategy: {}
+          template:
+            metadata:
+              annotations:
+                kubectl.kubernetes.io/default-container: manager
+              labels:
+                control-plane: controller-manager
+            spec:
+              containers:
+              - args:
+                - --secure-listen-address=0.0.0.0:8443
+                - --upstream=http://127.0.0.1:8080/
+                - --logtostderr=true
+                - --v=0
+                image: gcr.io/kubebuilder/kube-rbac-proxy:v0.16.0
```

Comment:
Do we need that?
In any case, kube-rbac-proxy is depreacted.
See https://github.com/Mellanox/network-operator/issues/1201